.github/.codecov.yml

```diff
@@ -0,0 +1,5 @@
+coverage:
+  status:
+    patch:
+      default:   # default is the status check's name, not default settings
+        target: 0 # never fail - coverage is informational only so far.
```

Comment:
That just removes default expectations for some minimum coverage by codecov. I don't want it to be in our way for now, I just like the info!